.github/workflows/regression-tests.yml

```diff
@@ -19,25 +30,35 @@ jobs:
         uses: actions/checkout@v4
         with:
           repository: 0xPolygon/kurtosis-cdk
-          ref: main
+          ref: "v0.2.15"
           path: kurtosis-cdk
 
-      - name: Install Kurtosis CDK tools
-        uses: ./kurtosis-cdk/.github/actions/setup-kurtosis-cdk
+      - name: Install kurtosis
+        shell: bash
+        run: |
+          echo "deb [trusted=yes] https://apt.fury.io/kurtosis-tech/ /" | sudo tee /etc/apt/sources.list.d/kurtosis.list
+          sudo apt update
+          sudo apt install kurtosis-cli=1.3.0
+          kurtosis version
+  
+      - name: Disable kurtosis analytics
+        shell: bash
+        run: kurtosis analytics disable
+  
+      - name: Install yq
+        shell: bash
+        run: |
+          pip3 install yq
+          yq --version
 
       - name: Build docker image
         working-directory: ./cdk-data-availability
         run: docker build -t cdk-data-availability:local --file Dockerfile .
 
-      - name: Configure Kurtosis CDK
-        working-directory: ./kurtosis-cdk
-        run: |
-          yq -Y --in-place '.args.data_availability_mode = "cdk-validium"' params.yml
-          yq -Y --in-place '.args.zkevm_da_image = "cdk-data-availability:local"' params.yml
-
       - name: Deploy Kurtosis CDK package
         working-directory: ./kurtosis-cdk
-        run: kurtosis run --enclave cdk-v1 --args-file params.yml --image-download always .
+        run: >
+          kurtosis run --enclave cdk . '{"args": {"zkevm_da_image": "cdk-data-availability:local"}}'
```

Comment:
We need to provide the `consensus_contract_type` variable as well (https://github.com/0xPolygon/kurtosis-cdk/blob/473422d4a02d978de2b61199e444fd7d0c98f7f6/input_parser.star#L297). It should be the `cdk-validium`.

Reply:
isn't it already `cdk-validium` by default?

Reply:
Yes, but I was thinking, maybe it is better to be explicit (in case that default value changes). WDYT?